_data/sample.yml

```
@@ -268,6 +272,22 @@ categories:
             url: https://jagat-xpub.github.io/cosmology/
           - name: *link_name_source
             url: https://github.com/jagat-xpub/cosmology
+  - name:
+      ja: 図版の多い横組書籍
+      en: Horizontal books with a lot of figures
+    samples:
+      - name:
+          ja: 海外の旅〔上〕
+          en: “Travel Abroad (The first volume)”
```

```suggestion
          en: “Travels Abroad (Volume 1)”
```

複数の旅の記録なので複数形Travelsに。